app/commands/markdown/preprocess.rb

```diff
@@ -0,0 +1,9 @@
+class Markdown::Preprocess
```

Comment:
I've not added tests for this particular command as it is indirectly tested via the tests for the `Markdown::Parse` command. The same goes for the other extracted commands

Reply:
Cool. I might add these when I do the styling.